### PR TITLE
Allow building without std on nightly

### PR DESCRIPTION
**Pull request by thomwiggers**

### CODE REVIEW

**Comment by dstebila** [comment]
@thomwiggers, is this ready to merge?

**Comment by thomwiggers** [comment]
Probably. There's probably nobody who I should ask to review, right? (fortunately we have decent CI)

File: oqs/src/lib.rs [hunk 1]
```diff
@@ -51,14 +55,26 @@ mod macros;
 ///
 /// Make sure to call this before you use any of the functions.
 ///
-/// This method is thread-safe and can be called more than once.
+/// When the default `std` method is enabled, this method is thread-safe
```
Comment (thomwiggers) [inline oqs/src/lib.rs:58]:
```suggestion
/// When the `no_std` feature is not enabled, this method is thread-safe
```

**Comment by thomwiggers** [comment]
Ah, I also can't merge myself with how this repository is set up.

**Comment by dstebila** [comment]
> Ah, I also can't merge myself with how this repository is set up.

I've given you admin access so you can override merge requirements in the future.

**Pull request merged**